CMakeLists.txt

```diff
@@ -2,9 +2,125 @@
 # auxil/zeek-aux/plugin-support/skeleton/CMakeLists.txt
 cmake_minimum_required(VERSION 3.15.0 FATAL_ERROR)
 
+if ( WIN32 )
```

Comment:
```suggestion
if ( MSVC )
```

Reply:
Can't do, MSVC being registered only after invocation to the `project()` function